RELEASE.md

```diff
@@ -234,6 +235,20 @@ Patch releases should be considered if a regression meets the following criteria
 3. Triage reviewers will then add the issue / pull request to the related milestone (i.e. `1.9.1`) if the regressions if found to be within the [Patch Release Criteria](#patch-release-criteria)
     * ![adding to milestone](https://user-images.githubusercontent.com/1700823/131175980-148ff38d-44c3-4611-8a1f-cd2fd1f4c49d.png)
 
+### Issue Tracker for Patch releases
+
+For patch releases issue tracker needs to be created. An example of this would look like:
```

Comment:
```suggestion
For patch releases issue tracker needs to be created. For patch release, we require all cherry-pick changes to have links to either a high-priority Github issue or a CI failure from previous RC. An example of this would look like:
```